helm_deploy/pre-sentence-service/Chart.yaml

```diff
@@ -5,9 +5,9 @@ name: pre-sentence-service
 version: 0.2.0
 dependencies:
   - name: generic-service
-    version: 1.0.18
+    version: 1.6.0
     repository: https://ministryofjustice.github.io/hmpps-helm-charts
   - name: generic-service
     alias: gotenberg
-    version: 1.0.18
+    version: 1.6.0
```

Comment:
Note that the `gotenberg` ingress is disabled so no need to update there, just updating the version to the latest as good practice